.environment-scripts/install_local_packages.sh

```diff
@@ -8,6 +8,7 @@ local_packages_to_install=(
   external/vcm
   external/loaders
   external/fv3fit
+  external/fv3gfs-util
```

Comment:
I needed it for this line in my script:
```
STACK_DIMS = (fv3gfs.util.X_DIM, fv3gfs.util.Y_DIM, fv3gfs.util.TILE_DIM)
```
The reason we don't already have this dependency is because we currently hard-code these dimension names (especially "z") instead of using the module constants, which is prone to breaking if they change.

Reply:
Which script? Is this something in your explore repo? 

This change seems unrelated to the contents of the PR, it would be nice to keep this kind of things separate since it makes the review process more difficult.

Reply:
The change for me here was actually so my local development environment would have the right packages. I did an environment refresh and was then missing fv3gfs-util.

Reply:
Several of our packages should have this dependency and should import `fv3gfs.util.Z_DIM`. Should I do those together in another PR, or should I hard-code dimension names in my user script instead?

Reply:
Conclusion was for now to add it to the environment, since it doesn't add dependencies we aren't already using.